src/Test/QuickCheck/StateMachineModel/Example.hs

```diff
@@ -104,7 +104,7 @@ semStep prb (Inc ref)     = liftIO $ do
 
 ------------------------------------------------------------------------
 
-debugMem :: MonadIO io => [Untyped MemStep (Ref, Int)] -> io ()
+debugMem :: MonadIO io => [Untyped MemStep (Ref, Pid)] -> io ()
```

Comment:
Hmm I would like to move this file out from the example, make it take `semStep` as a parameter. Probably not in this PR, but we should really start to split up our files, and have the example only contain code that is specific for the example (the shorter the better).